net-firewall/arno-iptables-firewall/arno-iptables-firewall-2.1.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit readme.gentoo-r1 systemd
+
+DESCRIPTION="Arno's iptables firewall script"
+HOMEPAGE="https://rocky.eld.leidenuniv.nl"
+
+MY_PV=$(ver_rs 3 -)
+MY_PV=${MY_PV/rc/RC}
+SRC_URI="https://github.com/${PN}/aif/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+plugins rsyslog"
+
+# sys-apps/coreutils dependency wrt
+# https://bugs.gentoo.org/show_bug.cgi?id=448716
+
+DEPEND=""
+RDEPEND="net-firewall/ipset
+	net-firewall/iptables
+	>=sys-apps/coreutils-8.21
```

Comment:
This is always satisified in ::gentoo. You can remove the version restriction, and the comment above about coreutils.

Reply:
Yup, that's seems like a leftover from older times, I wondered when it would be best to remove. I'll do it now.

Reply:
Yup, that's seems like a leftover from older times, I wondered when it would be best to remove. I'll do it now.

Reply:
Removed the coreutils bits, then rebased on top of current master and force pushed the updated version.